library/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+pub const CORE_LIB: &[(&str, &str)] = &[
```

Comment:
This seems like an opportunity to simplify the lookup in [`wasm/src/lib.rs`](https://github.com/microsoft/qsharp/blob/1aae38c481d14a50567ca8e73b4e8cea8318f9df/wasm/src/lib.rs#L40) . Would it be worth the effort to refactor?

Reply:
I think we could do this as another PR, but I think it is beyond what we're trying to accomplish here.